### DISCUSSION TITLE
部分字体库检测时显示为零

### DISCUSSION THREAD

**Discussion by mustang88** [NightFurySL2001/CJK-character-count#23] [Q&A]
感谢您提供的这个字体数检测软件，非常实用！

中华书局宋体及其他个别字库，在检测时会显示为0，即各个子项都显示为0。不知道是什么原因。
https://www.ancientbooks.cn/helpcore?font

**Comment by NightFurySL2001** [comment]
请检查该字体是否包含汉字，且不是私用区字符。“中华书局宋体”只有 Plane 0 和 2 包含正式汉字，Plane 15是私用区字符所以不计算。
